docs/b2b-edition/specs/storefront/storefront/sales-rep.yaml

```diff
@@ -1,499 +1,370 @@
-                      addressLine2: address line 2
-                      companyStatus: '1'
-                      city: Denver
-                      state: ''
-                      zipCode: '80022'
-                      country: Aruba
-                      companyId: '5716'
-                      catalogId: '5'
-                      extraFields:
-                        - fieldName: test
-                          fieldValue: test
-      operationId: get-sales-reps-customerId-companies-masquerading
-      description: |-
-        Get company information for the current super admin masquerading.
-        Equivalent Storefront GraphQL API Query: `superAdminMasquerading`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
+                allOf:
+                  - $ref: https://raw.githubusercontent.com/bigcommerce/docs/main/docs/b2b-edition/models/response/code-object-message.yaml
+                  - properties:
+                      message:
+                        default: SUCCESS
+                      code:
+                        default: 200
+                      data:
+                        properties:
+                          email:
+                            type: string
+                            format: email
+                            description: "The email address associated with the Super Admin account."
+                            example: "marie@example.com"
+                          firstName:
+                            type: string
+                            description: "The first name associated with the Super Admin account in B2B Edition."
+                            example: "Marie"
+                          lastName:
+                            type: string
+                            description: "The last name associated with the Super Admin account in B2B Edition."
+                            example: "Curie"
+                          phoneNumber:
+                            type: string
+                            description: "The phone number associated with the Super Admin account in B2B Edition."
+                            example: 9876543210
+  /sales-reps/{superAdminId}/companies/{companyId}/end-masq:
+    parameters:
+      - $ref: "#/components/parameters/superAdminId"
+      - $ref: "#/components/parameters/companyId"
+    put:
       tags:
         - Super Admin
-  '/sales-reps/{salesRepId}/companies':
-    parameters:
-      - schema:
-          type: string
-        name: salesRepId
-        in: path
-        required: true
-        description: Unique ID of super admin
-    get:
-      summary: Get Company List by Super Admin
+      summary: "End a Super Admin Masquerade"
+      operationId: put-sales-reps-customerId-companies-companyId-end-masq
+      description: |-
+        Updates the status of the Super Admin associated with `superAdminId`, ending their masquerade as the Company associated with `companyId`.
```

Comment:
This begs the question of why we need to specify a `companyId` at all. What is the result if a different `companyId` is specified than the one the Super Admin is currently masquerading as?